.babelrc

```diff
@@ -1,5 +1,5 @@
 {
-  "presets": ["react", "es2015"],
+  "presets": ["react", "es2015", "stage-2"],
```

Comment:
Allows us to use object spread provided in [current es-next Stage 3](https://github.com/hemanth/es-next#rest-and-spread-properties).

```
let a = {'one': 1};
let b = {'two': 2};
let c = {...a, ...b};
console.log(c);
// outputs: {'one': 1, 'two': 2}
```